cli/src/semdep/find_lockfiles.py

```diff
@@ -28,92 +22,6 @@ class Node:
     val: Optional[Dict[Path, List[LockfileDependency]]]
 
 
-class DependencyTrie:
```

Comment:
@mmcqd I think with the approach we took to get lockfiles we don't need this Trie anymore?